### PR TITLE
Add example for headers and link in the docs

### PR DESCRIPTION
**Pull request by anteloio**
Ref #15073

Thanks @lucavictor220 for the pairing session and @leerob for being patient as hell ❤️

### CODE REVIEW

File: examples/headers/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Redirects Example
```
Comment (leerob) [inline examples/headers/README.md:1]:
```suggestion
# Headers Example
```

File: examples/headers/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "redirects",
```
Comment (leerob) [inline examples/headers/package.json:2]:
```suggestion
  "name": "headers",
```

**Review by leerob — APPROVED** [review]
Great job! 🔥

**Comment by fwuensche** [comment]
> 🛑  cannot merge (missing required reviews)

Hi there. Do I need another approval after updating the branch?

**Review by leerob — APPROVED** [review]

**Pull request merged**